src/js/controllers/PromoControllers.js

```diff
@@ -106,7 +106,7 @@ angular.module('blocktrail.wallet')
                 .then(function(address) {
                     $scope.promoCodeInput.address = address;
                     trackingService.trackEvent(trackingService.EVENTS.PROMO_ATTEMPT);
-                    return activeWallet.getSdkWallet().sdk.redeemPromoCode($scope.promoCodeInput);
+                    return activeWallet.getWalletSdk().sdk.redeemPromoCode($scope.promoCodeInput);
```

Comment:
I think best name would be `getSdkWallet` since it's the SDK's Wallet object

Reply:
`getSdkWallet` -> return the `Wallet` object from the SDK (`self._sdkWallet` in the service) 
`getSdk` (or `getWalletSdk`) -> return the `self._sdkWallet.sdk`